docs/support/troubleshooting.md

```diff
@@ -141,8 +141,7 @@ When running the Planning Simulator, the most common reason for the map not bein
 
 Another possible reason is that map loading is taking a long time due to poor DDS performance. To address this issue, first enable localhost-only communication to reduce network traffic, and then [tune DDS settings](https://docs.ros.org/en/rolling/How-To-Guides/DDS-tuning.html) if the problem continues to occur.
 
-1. [Enable localhost-only communication](https://autowarefoundation.github.io/autoware-documentation/main/installation/tools-for-developers/#enabling-localhost-only-communication)
-
+1. Enable localhost-only communication
```

Comment:
Instead of removing the dead link, can we update it to point at the correct location instead?

@kenji-miyake I originally put the full URL in, but that would break for anyone not looking at main, so I assume that a relative link is better?

```suggestion
1. [Enable localhost-only communication](../../installation/additional-settings-for-developers/#enabling-localhost-only-communication)
```

Reply:
Yeah, exactly!